ontobio/sparql/sparql_ontology.py

```diff
@@ -122,9 +122,9 @@ def resolve_names(self, names, is_remote=False, synonyms=False, **args):
             results = set()
             for name in names:
                 results.update( self._search(name, 'rdfs:label', **args) )
-            if synonyms:
-                for pred in OIO_SYNS.values():
-                    results.update( self._search(name, pred, **args) )
+                if synonyms:
```

Comment:
It appears that there may be TAB characters here instead of SPACEs. Please check.